src/components/pages/ImageClassification/ImageClassificationTable/ImageClassificationObservationTable.tsx

```
@@ -0,0 +1,738 @@
+  ImageWrapper,
+  LoadingTableBody,
+  Spinner,
+  StyledTd,
+  StyledTr,
+  TdWithHoverText,
+} from './ImageClassificationObservationTable.styles'
+import Thumbnail from './Thumbnail'
+import { ImageClassificationImage } from '../../../../types/ImageClassificationTypes'
+import getObservationValidationInfo from '../../collectRecordFormPages/CollectRecordFormPage/getObservationValidationInfo'
+import ObservationValidationInfo from '../../collectRecordFormPages/ObservationValidationInfo'
+import { MessageType } from '../../../../theme'
```

ESLint

[error] MessageType not found in '../../../../theme'

(import/named)